components/mediation-admin/org.wso2.carbon.message.processor/src/main/java/org/wso2/carbon/message/processor/service/MessageProcessorAdminService.java

```diff
@@ -870,4 +876,12 @@ private MessageConsumer getMessageConsumer(SynapseConfiguration configuration, S
         String messageStoreName = processor.getMessageStoreName();
         return configuration.getMessageStore(messageStoreName).getConsumer();
     }
+
+    //TODO : Name probably needs to be changed
+    // Is the functionality a repetition of handleException?
+    private AxisFault returnAxisFaultException(Log log, String message, Exception e) throws AxisFault {
```

Comment:
Would the method name 'createAxisFaultException' be better?

Reply:
ill change it to createAxisFaulltException